pkg/auth/authz_utils.go

```diff
@@ -0,0 +1,29 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/openshift/assisted-service/internal/common"
+	"github.com/openshift/assisted-service/pkg/ocm"
+	"github.com/pkg/errors"
+)
+
+const (
+	NoAccessToMultiarch = "multiarch clusters are not available"
+)
+
+func ValidateAccessToMultiarch(ctx context.Context, authzHandler Authorizer) error {
```

Comment:
Not sure we need this new utils file. If we pass authzHandler to versions any way, we can add this func to the handler. That way we could reuse the same cache. Also, please add unit-tests to this func.

Reply:
+1
This is how we did if for `V2ListSupportedOpenshiftVersions` in https://github.com/openshift/assisted-service/pull/3814/

Reply:
`ValidateAccessToMultiarch` should be a part of `versionsHandler`.

Reply:
>Also, please add unit-tests to this func.

I may be missing something, but the only thing this function does is calling `authzHandler.HasOrgBasedCapability` and blindly parsing the result (a touple of boolean and error). So unit test would be here something like

```
result, error := true, nil
if result == true {
    do something
} else {
    do something else
}
```

>Not sure we need this new utils file. If we pass authzHandler to versions any way, we can add this func to the handler
>ValidateAccessToMultiarch should be a part of versionsHandler.

ACK, will do and move. Maybe we should have a generic utility library for handling stuff around `authzHandler.HasOrgBasedCapability`? I'm the second person doing org-based capability and the first implementation was different from the one I'm going to do. This sounds like a something that should be potentially "frameworkable"

Reply:
Right, some refactoring for that would be nice. Need to think how to structure it, maybe some lib just for capability checks:/ About test, it'd indeed should be a very simple test, don't you think it worth it? I mean, just to ensure we handle the error correctly and coverage..

Reply:
I think it's worth in the function of increasing our coverage. I think it's not worth in the function of usefulness :) I'll add it as it's indeed not difficult so no reason not to do it

Reply:
@danielerez unit test in place, I think we are complete now